src/views/List.jsx

```diff
@@ -11,6 +12,12 @@ export function List({ data }) {
 	};
 
 	useEffect(() => {
+		console.log(
+			'incoming data',
+			filteredList,
+			Boolean(filteredList),
+			Boolean(filteredList.length),
+		);
```

Comment:
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( if-minor ):** Extra console log here! Let's clean that up before merging :)